src/js/utils/specs/date-utils-spec.js

```diff
@@ -0,0 +1,145 @@
+define(
+    [
+        '../../utils/spec-helpers/spec-helpers.js',
+        '../date-utils.js',
+        'moment',
+        'moment-timezone'
+    ],
+    function(
+        SpecHelpers,
+        DateUtils,
+        moment,
+        momentTZ
+    ) {
+        'use strict';
+
+        describe('DateUtils', function() {
+            beforeEach(function() {
+                setFixtures('<div class="test"></div>');
+            });
+
+            describe('stringToMoment', function() {
+                it('returns an empty string if provided no time', function() {
+                    expect(DateUtils.stringToMoment()).toEqual('');
+                });
+                it('will convert a string to an moment.js object', function() {
+                    expect(
+                        DateUtils.stringToMoment('2016-10-14 08:00:00+00:00').format('ll')
+                    ).toEqual('Oct 14, 2016');
+                });
+            });
+
+            describe('getSysTimezone', function() {
+                it('attempts to determine users timezone', function() {
+                    // we need to determine the system tz offset from which to compare
+                    var sysDate = new Date();
+                    var sysDateOffset = sysDate.getTimezoneOffset();
+                    var dateUtilOffset = momentTZ.tz.zone(DateUtils.getSysTimezone()).offset(sysDate);
+                    expect(sysDateOffset).toEqual(dateUtilOffset);
+                });
```

Comment:
A contract of this public method is that it "falls back to UTC".  Did you plan to add a test to make sure that it does?